src/coreclr/gc/gc.cpp

```diff
@@ -7498,6 +7574,34 @@ bool gc_heap::is_in_condemned (uint8_t* o)
     else
         return false;
 }
+
+inline
+bool gc_heap::should_check_brick_for_reloc (uint8_t* o)
+{
+    assert ((o >= g_gc_lowest_address) && (o < g_gc_highest_address));
+
+    int condemned_gen = settings.condemned_generation;
+    if (condemned_gen < max_generation)
+    {
+        heap_segment* region = region_of (o);
+        int gen = get_region_gen_num (region);
+        if ((gen > condemned_gen) || (heap_segment_swept_in_plan (region)))
+        {
+            if (heap_segment_swept_in_plan (region))
+            {
+                dprintf (4444, ("-Rsip %Ix", o));
```

Comment:
is `4444` to signify something special? Everywhere else I am noticing its `REGIONS_LOG`

Reply:
it means it's a very verbose level which I didn't want to enable for REGIONS_LOG. I can change it to something more meaningful, perhaps REGIONS_LOG_VERBOSE.

Reply:
yeah a constant might be helpful, can happen as part of next change however.